scipy/signal/filter_design.py

```diff
@@ -1033,6 +1033,26 @@ def zpk2sos(z, p, k, pairing='nearest'):
     return sos
 
 
+def _align_nums(nums):
```

Comment:
A minimal docstring here would be helpful. Looking at the code it's not immediately obvious why the `try/except` is necessary and what it's doing